### DISCUSSION TITLE
MacOS support

### DISCUSSION THREAD

**Discussion by klutchdev** [supabase/supabase#3324] [Questions] [answered]
Hey everyone, I was just curious if the Supabase Flutter/Dart package has MacOS/Desktop support?
I know it's still in development, so it's not a huge deal - I'd just love to bang out a Supabase/Flutter  app for desktop and run it on my m1 mac mini!
Haha cheers everyone, keep up the awesomeness!

**Comment by dshukertjr** [comment, marked as answer]
Hi @klutchdev!

What specific features are you looking to use with this app? `supabase_flutter` supports most features on desktops except deep links. We use an external plugin called [uni_links](https://pub.dev/packages/uni_links) to handle deep links, which currently does not have desktop support. Otherwise, everything else should work just fine on desktop app as well!

**Reply by klutchdev** [reply]
Thanks pal!!
Honestly, I'm not worried about auth stuff for this one, I just wanna hook it up to use postgres!

**Reply by elliottetzkorn** [reply]
Any idea when desktop support for deep links will arrive? This blocks social sign in correct?

**Comment by k0shk0sh** [comment]
I used `app_links` instead of uni_links and override `startDeeplinkObserver` like below: 

```
@override
  void startDeeplinkObserver() {
    if (Platform.isAndroid || Platform.isIOS) {
      return super.startDeeplinkObserver();
    } else {
      _streamSubscription = _appLinks.uriLinkStream.listen((event) {
        handleDeeplink(event);
      });
    }
  }
```

worked like a charm and i'm able to provide supabase auth in desktop and mobile.
